set.mm

```diff
@@ -629568,10 +629546,12 @@ standardize vectors to a length (norm) of one, but such definitions require
   ${
     $d x y $.
     $( Version of ~ hbsb2 with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.)  (Proof
+       shortened by BJ, 11-Jan-2024.)  (Revised by Wolf Lammen,
+       12-Jan-2024.) $)
     hbsb2w $p |- ( -. A. x x = y -> ( [ y / x ] ph -> A. x [ y / x ] ph ) ) $=
-      ( weq wal wn wsb wi sb4bw sb2w axc4i syl6bi ) BCDZBEFABCGZMAHZBENBEABCION
-      BABCJKL $.
+      ( weq wal wn wsb wi wb sb6 a1i biimpri axc4i syl6bi ) BCDZBEFZABCGZOAHZBE
+      ZQBEQSIPABCJZKRQBQSTLMN $.
```

Comment:
Replace this with hbs1.

More generally, if you see an antecedent `|- ( -. A. x x = y -> ` then the version with DV(x,y) will not need that antecedent.

Reply:
I'm preparing a new PR to remove the distinctors from my lemmas (and delete hbsb2w). Probably I'm going to publish it tomorrow (I won't add the commit here).

Reply:
So: do you want this PR merged as is ? or deleted ?  (Fine by me either way.)

Reply:
Merge as is. This takes a bit of time because I have make some revisions to [ax-13-complete](https://github.com/GinoGiotto/set.mm/tree/ax-13-complete)